### PR TITLE
Updating segment properties and regressions

### PR DESCRIPTION
**Pull request by mclarke2**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

File: regression/scripts/concorde/concorde.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def main():
     results.fuel_tank_test.cg_maxes = cg_maxes
     
     # load older results
-    #save_results(results)
+    save_results(results)
```
Comment (planes) [inline regression/scripts/concorde/concorde.py:88]: comment this out

Comment (mclarke2) [reply regression/scripts/concorde/concorde.py:88]: commented out

Comment (mclarke2) [reply regression/scripts/concorde/concorde.py]: this has been addressed a while ago

**Review by planes — APPROVED** [review]
Github has been giving me problems viewing this. So I can't guarantee it will give you a comment I didn't intend.

**Pull request merged**